HoloPipelines/pipelineServer.py

```diff
@@ -0,0 +1,201 @@
+from flask import Flask, request
+from pipelineController import startPipeline, getPipelineList
+from flask_json import json_response
+from datetime import datetime
+from pathlib import Path
+import uuid
+import json
+import threading
+import time
+import logging
+import requests
+import os
+import pathlib
+from zipfile import ZipFile
+
+app = Flask(__name__)
+app.config["JSON_ADD_STATUS"] = False
```

Comment:
because I use the flask_json library for creating json response to the front end and in default it will generate a status code by default but the status code i want to generate is not the default value. for example in default  library will return a json response to the request with status code 200 but i want to send 202 instead.

Reply:
Ohh okay. Please add flask_json to the requirements.txt then, along with other dependencies. 

Btw, if i understand the docs correctly, it will just include the status code _that you set_, not always the default value. If you do `return json_response(jobID=jobID, status_code=202)`, it should include 200. But it's fine to just turn the feature off, I guess.